test/Ocelot.AcceptanceTests/RequestIdTests.cs

```diff
@@ -171,7 +171,7 @@ private void GivenThereIsAServiceRunningOn(string url)
             _serviceHandler.GivenThereIsAServiceRunningOn(url, context =>
             {
                 context.Request.Headers.TryGetValue(_steps.RequestIdKey, out var requestId);
-                context.Response.Headers.Add(_steps.RequestIdKey, requestId.First());
+                context.Response.Headers[_steps.RequestIdKey] = requestId.First();

```

Comment:
Not sure it should be changed for .NET 8 upgrade... You can include this change in current open PRs

Reply:
Same as before.

Reply:
Not an issue